azure-pipelines.yml

```diff
@@ -197,7 +215,7 @@ jobs:
         push daml-script-runner $(script-runner)
         push daml-script-runner $(script-runner).asc
       env:
-        AUTH: $(ARTIFACTORY_USERPASS)
+        AUTH: $(ARTIFACTORY_USERNAME):$(ARTIFACTORY_PASSWORD)
```

Comment:
we probably want to keep this and delete the rest now that we have an actual oracle job that uses the credentials.

Reply:
👍